src/reducers/index.js

```diff
@@ -1,21 +1,31 @@
 import { INCREMENT, DECREMENT } from '../actions';
 
 const initialState = {
-  count: 0
-}
+	count : 0,
+};
 
 // Our reducer that handles our two action cases:
 // increment and decrement. It receives the state
 // of our redux store, along with an action created
 // by our action creator. What does the reducer
 // need to do with the count in each case?
 export default (state = initialState, action) => {
-  switch (action.type) {
-    case INCREMENT:
-    // Fill in the body of this case
-    case DECREMENT:
-    // Fill in the body of this case
-    default:
-      return state;
-  }
+	switch (action.type) {
+		case INCREMENT:
+			let updatedCount = state.count + 1;
+			return {
+				...state,
+				count : updatedCount,
+			};
+		// Fill in the body of this case
+		case DECREMENT:
+			// Fill in the body of this case
+			updatedCount = state.count - 1;
+			return {
+				...state,
+				count : updatedCount,
+			};
```

Comment:
Really like how you abstracted `updatedCounted ` , makes everything a lot cleaner.